### PR TITLE
Remove `Bundler.setup` code

### PR DESCRIPTION
**Pull request by ramontayag**
It does not (perhaps no longer?) shows up when doing `bundle gem newgem`

### CODE REVIEW

File: Rakefile [hunk 1]
```diff
@@ -1,4 +1 @@
-require "bundler/gem_tasks"
```
Comment (nebolsin) [inline Rakefile]: Are you sure about removing this line? Without it we no longer have `rake build`, `rake install`, and a few other rake tasks.

Comment (ramontayag) [reply Rakefile]: No, I don't want to break that. Thanks!

File: ruby-stellar-sdk.gemspec [hunk 2]
```diff
@@ -24,7 +24,6 @@ Gem::Specification.new do |spec|
   spec.add_dependency "activesupport", ">= 4.2.7"
   spec.add_dependency "toml-rb", "~> 1.1", ">= 1.1.1"
 
-  spec.add_development_dependency "bundler", "~> 1.7"
```
Comment (nebolsin) [inline ruby-stellar-sdk.gemspec:27]: I think the dependency is still there with the latest Bundler 1.16.2

```console
$ bundle -v
Bundler version 1.16.2

$ bundle gem newgem > /dev/null && grep bundler newgem/*.gemspec
  spec.add_development_dependency "bundler", "~> 1.16"
```

**Comment by ramontayag** [comment]
You're right - I could have sworn I checked it and didn't see it. Thanks
again. Will fix.

**Comment by nullstyle** [comment]
👍 Thanks!

**Pull request merged**